maxim/logger/livekit/worker.py

```
@@ -39,18 +53,26 @@ def intercept_on(self:Worker, *args, **kwargs):
-        scribe().info(f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker started")
+        scribe().info(
+            f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker started"
+        )
     elif action == "worker_stopped":
-        scribe().info(f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker stopped")
+        scribe().info(
+            f"[Internal][MaximSDK][LiveKit:{self.__class__.__name__}] Worker stopped"
+        )
 
 
 def intercept_aclose(self, *args, **kwargs):
```

Ruff (0.11.9)

Missing return type annotation for public function `intercept_aclose`

Add return type annotation: `None`

(ANN201)